client/me/security-2fa-code-prompt/index.jsx

```diff
@@ -217,13 +214,14 @@ export default localize(
 							autoFocus
 							className="security-2fa-code-prompt__verification-code"
 							disabled={ this.state.submittingForm }
-							name="verification-code"
```

Comment:
should we be checking that no other functionality directly depended on the name being "verification-code"?

Reply:
Yes, that's a real risk. For example, there could be CSS rules that depend on the name: `.field[name="verification-code"]`. But I didn't find anything.

Reply:
same as above: `<FormLabel htmlFor="verification-code">` may need to be changed

Reply:
`htmlFor` refers to the `id` attribute, not `name`. So the value doesn't need to change. We could unify the naming for `id`s and `name`s, of course, but it's not strictly necessary.

Reply:
Ok, then feel free to ignore all my comments on `htmlFor`. I didn't find any other usages of the names you changed when I grepped about